app/services/exports/export_line_item_columns_adder.rb

```diff
@@ -0,0 +1,73 @@
+module Exports
+  class ExportLineItemColumnsAdder
+    def initialize
+      @registry = []
+    end
+
+    def headers(item_headers)
```

Comment:
In general this code is written in an unusual manner... I'd need to see a bunch more comments here since I can't really understand what some of it is supposed to be doing. What does "registry" contain? What kind of "row" does `get_row` return? What `additional_columns` do we need and how does that relate to the `registry`...?